README.md

```diff
@@ -65,10 +65,14 @@ kubectl proxy &
 curl -XPOST -H 'Content-Type: application/json' http://localhost:8001/api/v1/namespaces/custom-metrics/services/custom-metrics-apiserver:http/proxy/write-metrics/namespaces/default/services/kubernetes/test-metric --data-raw '"300m"'
 ```
 
-```
+```bash
 # you can pipe to `jq .` to pretty-print the output, if it's installed
 # (otherwise, it's not necessary)
 kubectl get --raw "/apis/custom.metrics.k8s.io/v1beta1" | jq .
+# fetching certain custom metrics of namespaced resources
+kubectl get --raw "/apis/custom.metrics.k8s.io/v1beta1/namespaces/default/services/kubernetes/test-metric" | jq .
+# certain custom metrics of  non-namespaced resource
+kubectl get --raw "/apis/custom.metrics.k8s.io/v1beta1/nodes/node1/test-metric" | jq .
```

Comment:
In the test example, `curl -XPOST` only sets the value of  `/namespaces/default/services/kubernetes/test-metric`, which `nodes/node1/test-metric` should not be needed here
/cc @dgrisonnet

Reply:
> In the test example, `curl -XPOST` only sets the value of `/namespaces/default/services/kubernetes/test-metric`, which `nodes/node1/test-metric` should not be needed here /cc @dgrisonnet

done~